packages/client/src/Components/PaperEdits/PaperCutsPlayer/Player.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import VideoContext from 'videocontext';
+
+const getCanvas = (ref, width, height) => {
+  return (
+    <canvas
+      ref={ ref }
+      className='video-canvas'
+      width={ width }
+      height={ height }
+    >
+    </canvas>
+  );
+};
+
+class PaperCutsPlayer extends React.PureComponent {
+  constructor(props) {
+    const {
+      width = 640,
+      height = 360,
+      playlist = [],
+    } = props;
+
+    super(props);
+
+    this.playlist = playlist;
+    this.canvasRef = React.createRef();
+    this.canvas = getCanvas(this.canvasRef, width, height);
+  }
+
+  componentDidMount() {
+    this.videoContext = new VideoContext(this.canvasRef.current, console.error);
```

Comment:
seeing [`videocontext.js#L80`](https://github.com/bbc/VideoContext/blob/develop/src/videocontext.js#L80) probably best to pass an error message to `console.error` such as

```suggestion
    this.videoContext = new VideoContext(this.canvasRef.current, console.error('Failed to initialise VideoContext'));
```